### ISSUE TITLE
Any alternative library for tfa.losses.TripletSemiHardLoss() ?

### ISSUE THREAD

**Issue by mfide** [tensorflow/addons#2859]
Hi,

Since tfa is going to be down, I'm looking for an alternative/recent implementation for tfa.losses.TripletSemiHardLoss()?

Is there an overlapping library for the loss function?

Thank you.

**Comment by bhack** [comment]
You can try to ping again https://github.com/keras-team/keras-cv/issues/289
